whatsagent/settings.py

```diff
@@ -0,0 +1,148 @@
+"""
+Django settings for whatsagent project.
+
+Generated by 'django-admin startproject' using Django 4.0.3.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.0/ref/settings/
+"""
+import os
+from pathlib import Path
+
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+# SECRET_KEY = 'django-insecure-%qxd68yl5ya&2ct=_roi7o3u4z2v3jwa%3d7--)^)mh5an6yie'
+SECRET_KEY = os.getenv('SECRET_KEY')
+MAILGUN_API_KEY = os.getenv('MAILGUN_API_KEY')
+GMAIL_SENDER_APP_KEY = os.getenv('GMAIL_SENDER_APP_KEY')
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = []
+
+
+# Application definition
+
+INSTALLED_APPS = [
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    'rest_framework',
+
+    # 3rd party
+    'corsheaders',
+    'django_extensions',
+
+    # Project
+    'whatsagent.api',
+]
+
+MIDDLEWARE = [
+    'corsheaders.middleware.CorsMiddleware',
+    'django.middleware.security.SecurityMiddleware',
+    'django.contrib.sessions.middleware.SessionMiddleware',
+    'django.middleware.common.CommonMiddleware',
+    # 'django.middleware.csrf.CsrfViewMiddleware',
```

Comment:
You can remove `# 'django.middleware.csrf.CsrfViewMiddleware'`

Reply:
Removed CsrfViewMiddleware